rskj-core/src/main/java/co/rsk/rpc/modules/eth/EthModuleTransactionBase.java

```diff
@@ -41,85 +41,89 @@
+                TransactionArguments txArgs = TransactionArgumentsUtil.processArguments(args, transactionPool, senderAccount, constants.getChainId());
 
-				Transaction tx = Transaction.builder().withTransactionArguments(txArgs).build();
+                Transaction tx = Transaction.builder().withTransactionArguments(txArgs).build();
 
-				tx.sign(senderAccount.getEcKey().getPrivKeyBytes());
+                tx.sign(senderAccount.getEcKey().getPrivKeyBytes());
 
-				if (!tx.acceptTransactionSignature(constants.getChainId())) {
-					throw RskJsonRpcRequestException.invalidParamError(TransactionArgumentsUtil.ERR_INVALID_CHAIN_ID + args.getChainId());
-				}
+                if (!tx.acceptTransactionSignature(constants.getChainId())) {
+                    throw RskJsonRpcRequestException.invalidParamError(TransactionArgumentsUtil.ERR_INVALID_CHAIN_ID + args.getChainId());
+                }
 
-				TransactionPoolAddResult result = transactionGateway.receiveTransaction(tx.toImmutableTransaction());
+                TransactionPoolAddResult result = transactionGateway.receiveTransaction(tx.toImmutableTransaction());
 
-				if (!result.transactionsWereAdded()) {
-					throw RskJsonRpcRequestException.transactionError(result.getErrorMessage());
-				}
+                if (!result.transactionsWereAdded()) {
+                    throw RskJsonRpcRequestException.transactionError(result.getErrorMessage());
+                }
 
-				txHash = tx.getHash().toJsonString();
-			}
+                txHash = tx.getHash().toJsonString();
+            }
 
-			return txHash;
+            return txHash;
 
-		} finally {
-			LOGGER.debug("eth_sendTransaction({}): {}", args, txHash);
-		}
-	}
+        } finally {
+            LOGGER.debug("eth_sendTransaction({}): {}", args, txHash);
+        }
+    }
 
-	@Override
-	public String sendRawTransaction(String rawData) {
-		String s = null;
-		try {
-			Transaction tx = new ImmutableTransaction(HexUtils.stringHexToByteArray(rawData));
+    @Override
+    public String sendRawTransaction(String rawData) {
+        String s = null;
+        try {
+            Transaction tx = new ImmutableTransaction(HexUtils.stringHexToByteArray(rawData));
 
-			if (null == tx.getGasLimit() || null == tx.getGasPrice() || null == tx.getValue()) {
-				throw invalidParamError("Missing parameter, gasPrice, gas or value");
-			}
+            if (null == tx.getGasLimit() || null == tx.getGasPrice() || null == tx.getValue()) {
+                throw invalidParamError("Missing parameter, gasPrice, gas or value");
+            }
 
-			TransactionPoolAddResult result = transactionGateway.receiveTransaction(tx);
-			if (!result.transactionsWereAdded()) {
-				throw RskJsonRpcRequestException.transactionError(result.getErrorMessage());
-			}
+            if (!tx.acceptTransactionSignature(constants.getChainId())) {
+                throw RskJsonRpcRequestException.invalidParamError(TransactionArgumentsUtil.ERR_INVALID_CHAIN_ID + tx.getChainId());
```

Comment:
please cover this thrown exception in unit tests

Reply:
done